src/board.py

```diff
@@ -8,7 +8,7 @@ class Board:
     rank_to_index = {rank_: index_ for index_, rank_ in zip(range(8), "87654321")}
 
     def __init__(self):
-        self.matrix = [[None] * 8 for _ in range(8)]
+        self.board = [[None] * 8 for _ in range(8)]
```

Comment:
Imagine this:
```py
board = Board()
board.board[0][0]
```

I think the following would look better:
```py
board = Board()
board.matrix[0][0]
```
The board is the object and the `matrix` attribute is a matrix storing the state of the board.